pages.bs/common/mkfifo.md

```diff
@@ -1,6 +1,7 @@
 # mkfifo
 
 > Pravi FIFOs (imenovane cijevi).
+> ? <https://www.gnu.org/software/coreutils/manual/html_node/mkfifo-invocation.html>.
```

Comment:
👀 !

Reply:
@Vukory can we have the most accurate Bosnian translation of "More information:" ?

Reply:
Yeah, I'm searching for the correct translation at the moment :sweat_smile:

Reply:
Would `Više informacija` work?

Reply:
I think `Više informacija:` should be correct

Reply:
Let's do it.
```suggestion
> Više informacija: <https://www.gnu.org/software/coreutils/manual/html_node/mkfifo-invocation.html>.
```
I've also updated [`set_more_info_link.py`](https://gist.github.com/bl-ue/5b52c9f80dd21074c0633a2bcdeb01fb).

Reply:
Also, `da` is `Mere information:` and `no` is `Mer informasjon:`

Reply:
Oh, nice! I'll add those.

Reply:
Thanks!

Reply:
> Would `Više informacija` work?
`Više informacija` is perfect, though I see I'm a bit late sorry, been a rough day ^^'

Reply:
No problem, thank you very much for the confirmation! ❤️